### DISCUSSION TITLE
FilePicker.pickFiles() - File path url as "file://" and not only as "content://"

### DISCUSSION THREAD

**Discussion by renebruegger** [capawesome-team/capacitor-plugins#130] [Ideas]
Is there a chance to get the "path url" (File https://capawesome.io/plugins/file-picker/#pickfilesresult ) form the FilePicker.pickFiles() as "file://" url? 
Currently I only get a "content://" url and I have to convert it to "file" for further use.

**Comment by robingenz** [comment]
See https://github.com/capawesome-team/capacitor-plugins/issues/129#issuecomment-1905481710
